openroad_interface/test/test_ibexsky130hd.tcl

```diff
@@ -0,0 +1,234 @@
+# ibex sky130hd
```

Comment:
are these modified from the defaults at all? is there a reason we have a copy here? 
Or are these the scripts that you've modified to use the custom .def and ignore the .v

Reply:
update dir above to be `tcl`